### ISSUE TITLE
Usefulness 

### ISSUE THREAD

**Issue by greenamy808** [QERM514/assistance#23]
Hi, I have a question about homework 3 question c in explaining why the plot of residuals against the predictor variable "stage" is not useful. I suppose I'm wondering if it is because it's a category variable or because we've already determined that the variance fails the IID test?

**Comment by ameliaduvall** [comment]
Thanks for asking this! I wanted to make sure I was clear on this too. From what I remember in lecture, part of the reason is because stage is a categorical rather than a continuous variable, which I believe makes it more difficult to interpret this kind of plot.

**Comment by mdscheuerell** [comment]
Yes, both @greenamy808 & @ameliaduvall are onto the reasoning here. One thing to think about is the level of contrast in the predictor variable. Is there much variation in the *X* here?
